amida-tech/hl7-telescoper-client
# Dep up/react scripts and Linting

### Related JIRA tickets:
>Please enter the whole URL so we can just click/copy it
- None 😶

### What exactly does this PR do?
>i.e. Added logic to do the thing because the thing didn't exist before. Please provide as much detail as possible. 
- Upgrades `react-scripts` package
- Extends default create-react-app eslint rules with some amida rules
  - `semi: always`
  - `comma-dangle: always-multiline`
- adds two new yarn scripts
  - `yarn lint`: runs linter
  - `yarn lint:fix`: runs linter and fixes what can be automatically fixed
- I ran `yarn lint:fix` on the existing codebase and oh-so-many semicolons were added

### Do ANY of these changes introduce a breaking change? (in-scope or otherwise)
>i.e. Users need to update their `.env` files with the following... etc
- Nope

### Manual test cases?
>Steps to manually test introduced changes can go here.
>Remember the prerequisites!
>Remember edge-cases you've caught in your code, etc..
> i.e. ignoring the button and clicking on the "NEXT" button should trigger a warning event because the thing that clicking the new button does didn't happen
- introduce what should be a autofixable lint error, like a line without a semicolon
- run `yarn lint`, make sure you get a warning
- run `yarn lint:fix` and make sure it fixes the problem
- review the rules I added, are there any other important amida-style rules we should add in this PR?

### Any additional context/background?
>N/A if this is straight-forward
- Specifically the lack of semicolon consistency was getting to me (and I committed a lot of both semicolonful and semicolonless code, so not blaming anyone but myself)

### Screenshots (if appropriate):
N/A
